### PR TITLE
disttask: storage.util move to testutil

### PR DESCRIPTION
**Pull request by ywqzzy**
### What problem does this PR solve?

Issue Number: ref #48795

Problem Summary:
As the title said.
### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by okJiang — APPROVED** [review]

File: pkg/disttask/framework/testutil/table_util.go [hunk 1]
```diff
@@ -74,3 +79,132 @@ func getTaskManager(t *testing.T, pool *pools.ResourcePool) *storage.TaskManager
...
+		 set exec_id = %?, state_update_time = unix_timestamp()
+		 where id = %?`,
+		tidbID, subtaskID)
+	return err
+}
+
+// PrintSubtaskInfo log the subtask info by taskKey for test.
+func PrintSubtaskInfo(ctx context.Context, mgr *storage.TaskManager, taskID int64) {
+	rs, _ := mgr.ExecuteSQLWithNewSession(ctx,
+		`select `+storage.SubtaskColumns+` from mysql.tidb_background_subtask_history where task_key = %?`, taskID)
```
Comment (D3Hunter) [inline pkg/disttask/framework/testutil/table_util.go:191]: actually we didn't query error field😓

**Review by D3Hunter — COMMENTED** [review]
rest lgtm

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**